mesonbuild/ast/interpreter.py

```diff
@@ -19,23 +19,31 @@
 from .. import interpreterbase, mparser, mesonlib
 from .. import environment
 
-from ..interpreterbase import InvalidArguments, BreakRequest, ContinueRequest
+from ..interpreterbase import InvalidArguments, BreakRequest, ContinueRequest, TYPE_nvar
 from ..mparser import (
+    AndNode,
     ArgumentNode,
     ArithmeticNode,
     ArrayNode,
     AssignmentNode,
     BaseNode,
+    ComparisonNode,
     ElementaryNode,
     EmptyNode,
+    ForeachClauseNode,
     IdNode,
+    IfClauseNode,
+    IndexNode,
     MethodNode,
+    OrNode,
     PlusAssignmentNode,
+    StringNode,
     TernaryNode,
+    UMinusNode,
```

Comment:
Does this impact performance in this case? I do need the other imports anyway. Also, I generally try to avoid forward declaring stuff, since VS code won't give me autocompletion support in this case :)

Reply:
vscode gives me autocomplete with forward declarations... Are you using jedi or the microsoft python langauge server? I use the latter.

Reply:
I am using the standard python extension `Python extension for Visual Studio Code`. Not sure if about the language server (if included in the extension).

I can do additional tests, but it seems like I only get autocomplete support if the classes are imported.